docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
+It is a MapReduce job that does distributed data loading from Kafka to HDFS.
+At LinkedIn, Camus is used to process tens of billions of messages per day.
+The confluent Camus distribution supports the latest CDH distribution.
+
+Camus includes the following features:
+    #. Automatic topic discovery
+    #. Avro schema management
+    #. Date partitioning
+
+All work is done within a single MapReduce job divided into three stages:
```

Comment:
I wouldn't even explain this here -- it's all implementation details, but the landing page for this should be more product-focused. I think this issue partly stems from the format I initially used to setup these templates, which is organized more for open source project READMEs.

We could keep this text, but I'd move it somewhere below quickstart.